lib/dts-lib.robot

```diff
@@ -0,0 +1,130 @@
+*** Settings ***
+Library     ../robot-venv/lib/python3.11/site-packages/robot/libraries/Collections.py
+
+
+*** Keywords ***
+Enable Network Boot
```

Comment:
We already have: https://github.com/Dasharo/open-source-firmware-validation/blob/main/dasharo-compatibility/network-boot.robot#L127

This kwd could be moved to some lib, such as bios/menus and used in multuple cases.

Please keep in mind, that some devices do not have this options (network boot is always enabled then). If this is the case, the test must still work.

Reply:
[Removed](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/44ad73e6c5e5fe00e30995617fa9463cd758d089#diff-0ca58a364903f744bc9458315dbf5ccc48c37f510ab670103bb67f789341868bL6) my keyword, copied the `network-boot.robot` keyword into [menus](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/44ad73e6c5e5fe00e30995617fa9463cd758d089#diff-6ad7c5fb97cb1099c7ecd42f15dde3caf06d38b9fe5244afb96394454b6d6bfaR508). Will remove it from `network-boot.robot` in a dedicated separate commit.